.github/workflows/ci-log-amend.yml

```
@@ -0,0 +1,144 @@
+name: CI with Log Amendment
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)